pkg/signoz/provider.go

```diff
@@ -72,6 +72,7 @@ func NewSQLMigrationProviderFactories(sqlstore sqlstore.SQLStore) factory.NamedM
 		sqlmigration.NewUpdateRulesFactory(sqlstore),
 		sqlmigration.NewAddVirtualFieldsFactory(),
 		sqlmigration.NewUpdateIntegrationsFactory(sqlstore),
+		sqlmigration.NewCreateQuickFiltersFactory(sqlstore),
```

Comment:
```suggestion
		sqlmigration.NewCreateQuickFiltersFactory(sqlstore),
```

Move this to the end of the list.